openmetadata-ui/src/main/resources/ui/src/pages/DatasetDetailsPage/DatasetDetailsPage.component.tsx

```diff
@@ -500,16 +518,16 @@ const DatasetDetailsPage: FunctionComponent = () => {
         })
         .catch((err: AxiosError) => {
           const msg =
-            err.response?.data.message || `Error while updating entity.`;
+            err.response?.data.message ||
+            jsonData['api-error-messages']['update-entity-error'];
+          handleShowErrorToast(msg);
           reject();
-          showToast({
-            variant: 'error',
-            body: msg,
-          });
         });
     });
   };
 
+  // TODO: move all the error from below code to en.ts and use handleShowErrorToast to show error.
```

Comment:
@ShaileshParmar11, Can you make the required changes for this entity page.

Reply:
It would be great if this will be handle in separate PR.